### PR TITLE
Update 05_node_operations.asciidoc

### PR DESCRIPTION
**Pull request by ghost**
Capitalization, formatting and minor rephrasing of one or two clauses to make them clearer. 

I'm working through the whole book, chapter by chapter, and so am trying to increase consistency.  For example, when terms are introduced, they should have the acronym in parenthesis the first time, not the second time.

### CODE REVIEW

File: 05_node_operations.asciidoc [hunk 1]
```diff
@@ -72,134 +72,134 @@ in the cloud. Nonetheless, if running your own server is not an option, by all m
...
-One of the most popular and complete "helper" is _RaspiBlitz_, a project built by Christian Rotzoll. It is intended to be installed on a Raspberry Pi 4. RaspiBlitz comes with a recommended hardware "kit" that you can build in a matter of hours or at most a weekend. If you attend a Lightning "hackathon" in your city, you are likely to see many people working on their RaspiBlitz setup, swapping tips, and helping each other. You can find the RaspiBlitz project here:
+One of the most popular and complete "helpers" is _RaspiBlitz_, a project built by Christian Rotzoll. It is intended to be installed on a Raspberry Pi 4. RaspiBlitz comes with a recommended hardware "kit" that you can build in a matter of hours or at most a weekend. If you attend a Lightning "hackathon" in your city, you are likely to see many people working on their RaspiBlitz setup, swapping tips, and helping each other. You can find the RaspiBlitz project here:
 
 https://github.com/rootzoll/raspiblitz
 
 image::images/raspiblitz.jpg[]
 
 In addition to a Bitcoin and Lightning node, RaspiBlitz can install a number of additional services, such as:
 
-* TOR (Run as Hidden Service)
```
Comment (ghost) [inline 05_node_operations.asciidoc:151]: The capitalization here is clearly copied from the RaspiBlitz website.  My suggested changes are more accurate.

File: 05_node_operations.asciidoc [hunk 2]
```diff
@@ -72,134 +72,134 @@ in the cloud. Nonetheless, if running your own server is not an option, by all m
...
+Famous for their UX/UI, _Umbrel_ provides a very easy and accessible way to get your Bitcoin and Lightning node up and running in no time, especially for beginners. A very distinctive feature is that _Umbrel_ utilizes Neutrino/SPV during the Inital Blockchain Download so you can instantly start using your node. Once Bitcoin Core is fully synced in the background it automatically switches over and disables SPV mode. Umbrel OS supports the Raspberry Pi 4 and can also be installed on any Linux-based OS or on a virtual machine on macOS or Windows. You can also connect any wallet that supports Bitcoin Core P2P, Bitcoin Core RPC, the Electrum protocol or lndconnect.
 
-No need to wait for a rainy day, you can find the project here: https://getumbrel.com
+There's no need to wait for a rainy day - you can find the project here: https://getumbrel.com
 
 image::images/umbrel.png[]
 
 In addition to a Bitcoin and Lightning node, Umbrel introduced the Umbrel App Store, where you can easily install additional services, such as:
 
-* Lightning Terminal (Interface for managing channel liquidity, loop-in & loop-out)
```
Comment (ghost) [inline 05_node_operations.asciidoc:188]: As above capitalization here is taken from another source.  My suggested changes are more accurate.

**Review by ghost — COMMENTED** [review]
I've suggested a number so small changes, mostly to do with capitalization.

File: 05_node_operations.asciidoc [hunk 3]
```diff
@@ -609,23 +609,23 @@ As an additional measure of security, you can make your second node a hidden TOR
...
 
-You could do this by trusting an intermediary to act as a gateway for your funds and not steal them. But in the case of a submarine swap, the operation does not require trust. Submarine swaps are non-custodial _atomic_ operations. That means that the counterparty in your submarine swap cannot steal your funds because the on-chain payment depends on the completion of the off-chain payment and vice-versa. We will discuss submarine swaps in more detail in <<submarine_swaps>>.
+You could do this by trusting an intermediary to act as a gateway, but this risks your fund being stolen. But in the case of a submarine swap, the operation does not require trust. Submarine swaps are non-custodial _atomic_ operations. That means that the counterparty in your submarine swap cannot steal your funds because the on-chain payment depends on the completion of the off-chain payment and vice-versa. We will discuss submarine swaps in more detail in <<submarine_swaps>>.
```
Comment (renepickhardt) [inline 05_node_operations.asciidoc]: fundS ?

**Review by renepickhardt — CHANGES_REQUESTED** [review]
think a typo was introduced. still large improvement. thanks!

**Pull request merged**